### DISCUSSION TITLE
Access to GitHub repo for pull requests

### DISCUSSION THREAD

**Discussion by eduardsmirnov** [OP-TED/eForms-SDK#487] [Q&A] [answered]
On the last workshop you said about a possibility to open a pull request if there seems something wrong in SDK. As of today there are no access to do it.

**Comment by bertrand-lorentz** [comment, marked as answer]
In order to create a pull request, you need to first fork the repository, make the changes, and then create a pull request from your repository to this one. See the GitHub documentation at: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

Here's an example of such a pull request that was made a while back:
https://github.com/OP-TED/eForms-SDK/pull/213

(I see this has been closed, I putting my answer above anyways in case it helps)
